src/state/slices/tradeInputSlice/tradeInputSlice.ts

```diff
@@ -53,6 +53,9 @@ export const tradeInput = createSlice({
       const isSameAsBuyAsset = asset.assetId === state.buyAsset.assetId
       if (isSameAsBuyAsset) state.buyAsset = state.sellAsset
 
+      // clear the sell amount
+      state.sellAmountCryptoPrecision = '0'
+
```

Comment:
deletions to `src/components/MultiHopTrade/components/TradeInput/components/SellAssetInput.tsx` are essentially regained with this line